driver/src/contracts/stablex_contract.rs

```diff
@@ -437,7 +413,7 @@ pub mod tests {
 
         // Only contain non fee-token and non 0 prices
         let expected_prices = vec![1.into(), 2.into()];
-        let expected_token_ids = vec![2.into(), 3.into()];
+        let expected_token_ids = vec![2, 3];
```

Comment:
Why does one need `into` and not the other?

Reply:
It used to be the `U128` non-primitive type and now its the `u64` primitive type.